ui/src/tasks/actions/index.ts

```diff
@@ -252,12 +267,18 @@ export const getTasks = () => async (
   }
 }
 
-export const addTaskLabelsAsync = (taskID: string, labels: Label[]) => async (
+export const addTaskLabelAsync = (taskID: string, label: Label) => async (
```

Comment:
renamed this to accurately reflect its functionality since only one label is ever passed